cargo-concordium/src/build.rs

```diff
@@ -419,6 +422,46 @@ fn write_schema_json(
     Ok(())
 }
 
+/// Write the provided schema in its base64 representation to a file inside the
+/// `root` directory if `Some(root)` is given.
+/// Print the provided schema in its base64 representation to the console if
+/// `schema_base64_log` is true.
+pub fn write_schema_base64(
+    root: Option<PathBuf>,
+    schema: &VersionedModuleSchema,
+    schema_base64_log: bool,
```

Comment:
Why is this necessary? I think we should just output to `stdout` if `root = None`.